custom_components/xiaomi_miot/camera.py

```diff
@@ -596,3 +598,153 @@ async def image_source(self, **kwargs):
         return await self.hass.async_add_executor_job(
             partial(self._parent.get_motion_image_address, **kwargs)
         )
+
+
+class VacummMapCameraEntity(BaseSubEntity, Camera):
+    def __init__(self, parent):
+        super().__init__(parent, 'vacuum_map')
+        self._should_poll = True
+        self._supported_features |= SUPPORT_ON_OFF
+        self.parent = parent
+        self._map_raw_data = None
+    @property
+    def access_tokens(self) -> str:
```

Comment:
https://github.com/home-assistant/core/blob/2b3df46f069cb4a85635023c4327a9a2a81a09b5/homeassistant/components/camera/__init__.py#L450

`access_tokens` is recommended to be managed by HA core.

Reply:
`entity_picture` should also be managed by HA core.
And your need init the base `Camera`: `Camera.__init__(self)` before [this line](https://github.com/al-one/hass-xiaomi-miot/blob/90cb64966937f10dcd5e7f9df09e7a831aac35c5/custom_components/xiaomi_miot/camera.py#L605).